google/mozc
# Filter out wrong pronunciation of 般若心経 in the dictionary

## Description

![image](https://github.com/user-attachments/assets/844a8c8a-82ad-4635-b5f2-468306d40ec6)

It is due to the fact that the wrong pronunciation for "般若心経" has been registered in [the dictionary](https://github.com/google/mozc/blob/master/src/data/dictionary_oss/dictionary06.txt) that the correction suggestion for "般若心経" appears even when you haven't typed "k" or "き" after "はんにゃしん".

![image](https://github.com/user-attachments/assets/a5d73fe3-c5c6-497e-b7e2-5c4987355a4a)

https://github.com/google/mozc/blob/ace314567109cbc30dc336fb27844dacff782dd9/src/data/dictionary_oss/reading_correction.tsv#L558

## Issue IDs
Issue and/or discussion IDs related to this pull request.

Fixes #1060

## Steps to test new behaviors (if any)
A clear and concise description about how to verify new behaviors (if any).
 - OS: any (I used Windows 11 Sandbox)

1. Type "はんにゃしん"
2. The correction suggestion hasn't appeared yet
3. Type "k"
4. It appears now

![image](https://github.com/user-attachments/assets/f72d9caf-5cae-4e1e-aa83-4d2113daf932)
![image](https://github.com/user-attachments/assets/b431d24f-6bd9-4886-8390-818a6917b415)
![image](https://github.com/user-attachments/assets/e8c99528-870f-452e-80d5-f4b2753937ca)



## Additional context
Add any other context about the pull request here.
